samigo/samigo-hibernate/src/java/org/sakaiproject/tool/assessment/data/dao/assessment/AssessmentBaseData.java

```diff
@@ -43,6 +43,7 @@
   static Category errorLogger = Category.getInstance("errorLogger");
 
   private static final long serialVersionUID = 7526471155622776147L;
+  public static final int title_length = 255;
```

Comment:
Please use CAPITALS for a contact declaration .... TITLE_LENGTH instead of title_length